tianshou/data/batch.py

```diff
@@ -128,12 +128,13 @@ def __repr__(self) -> str:
 
     def keys(self) -> List[str]:
         """Return self.keys()."""
-        return sorted([
-            i for i in self.__dict__ if i[0] != '_'] + list(self._meta))
+        return list(self._meta.keys()) + \
+            [k for k in self.__dict__.keys() if i[0] != '_']
 
     def values(self) -> List[Any]:
         """Return self.values()."""
-        return [self[k] for k in self.keys()]
+        return list(self._meta.values()) +
```

Comment:
I think the values in self._meta should not be returned?

Reply:
Yes, I fixed that.